docs/modules/Compactable.ts.md

```diff
@@ -231,20 +218,29 @@ Added in v2.0.0
 
 # utils
 
-## Separated (interface)
+## Separated
 
-A `Separated` type which holds `left` and `right` parts.
+Re-exported interface for backwards compatibility.
 
 **Signature**
 
 ```ts
-export interface Separated<A, B> {
-  readonly left: A
-  readonly right: B
-}
+export declare const Separated: any
```

Comment:
@gcanti I haven't found a way to re-export an interface in such way that `docs-ts` mark it as an interface.

Reply:
Ah that's a pity, let's keep the original definition then (it will be deprecated anyway)